db/seeds.rb

```diff
@@ -1,7 +1,8 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+FactoryGirl.create(:item)
+FactoryGirl.create(:item,
```

Comment:
Look into the "Configure your test suite" part of this page for ways to type less: 

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md